src/braket/aws/aws_quantum_task.py

```diff
@@ -92,23 +93,21 @@ class AwsQuantumTask(QuantumTask):
     def create(
         aws_session: AwsSession,
         device_arn: str,
-        task_specification: Union[
-            Circuit,
-            Problem,
-            OpenQASMProgram,
-            BlackbirdProgram,
-            PulseSequence,
-            AnalogHamiltonianSimulation,
-        ],
+        task_specification: Circuit
+        | Problem
+        | OpenQASMProgram
+        | BlackbirdProgram
+        | PulseSequence
+        | AnalogHamiltonianSimulation,
         s3_destination_folder: AwsSession.S3DestinationFolder,
         shots: int,
-        device_parameters: dict[str, Any] | None = None,
+        device_parameters: Optional[dict[str, Any]] = None,  # noqa: UP007
         disable_qubit_rewiring: bool = False,
-        tags: dict[str, str] | None = None,
-        inputs: dict[str, float] | None = None,
-        gate_definitions: dict[tuple[Gate, QubitSet], PulseSequence] | None = None,
+        tags: Optional[dict[str, str]] = None,  # noqa: UP007
+        inputs: Optional[dict[str, float]] = None,  # noqa: UP007
+        gate_definitions: Optional[dict[tuple[Gate, QubitSet], PulseSequence]] = None,  # noqa: UP007
         quiet: bool = False,
-        reservation_arn: str | None = None,
```

Comment:
This used to work, why not anymore?

Reply:
It still does, I am changing this back in the 3.10 pr